src/Altinn.Notifications.Integrations/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -53,6 +57,19 @@ public static void AddAltinnClients(this IServiceCollection services, IConfigura
         services.AddHttpClient<IRegisterClient, RegisterClient>();
     }
 
+    /// <summary>
+    /// Adds Altinn clients and configurations to DI container.
+    /// </summary>
+    /// <param name="services">service collection.</param>
+    /// <param name="config">the configuration collection</param>
+    public static void AddAuthorizationService(this IServiceCollection services, IConfiguration config)
+    {
+        services.Configure<Common.PEP.Configuration.PlatformSettings>(config.GetSection("PlatformSettings"));
```

Comment:
This fixed the error on my end

```suggestion
        services.Configure<Common.PEP.Configuration.PlatformSettings>(config.GetSection("PlatformSettings"));
services.AddSingleton<IHttpContextAccessor, HttpContextAccessor>();
```

Reply:
That is strange. The service is already being added in Program.cs line 206.